lib/helpers/afterScreenshot.interfaces.ts

```diff
@@ -21,11 +21,13 @@ export interface AfterScreenshotOptions {
   // The file name options object
   fileName: ScreenshotFileNameOptions;
   // Elements that need to be hidden (visibility: hidden) before saving a screenshot
-  hideElements: (HTMLElement | HTMLElement[])[];
+  hideElements: HTMLElement[];
```

Comment:
Why did you change the interface here? `hideElements` and `removeElements` will always be an array that hold or single elements, or a combination of single elements and an array of elements (like `[$('something'), $$('something-else')]`)